README.md

```diff
@@ -1,3 +1,36 @@
+S3L1M created a very useful program to help revert files captured by `ground.exe` virus.  I got infected, amd noticed it fairly quickly when I noticed `ground.exe` running in task manager.  I would like to create this updated readme with more detailed instructions for using Ground-Damage-Fixer. Of course, you already have `Ground` stopped and are stuck with this messy aftermath. I included how I removed `Ground.exe` below in case you came across this page while still searching for a way to remove the virus.
+
+The command to manualy unhide and retsore permissions to a file/folder is
+```
+attrib -H -S *.exe /S /D
+```
+
+1) Run `Super Unhider`
+2) Click on `Add Files`
+2) search for files with `size:534,016`
+![Alt Text](https://github.com/bizzar721/Ground-Damage-Fixer/blob/master/1111.png)
```

Comment:
Images should be stored in a folder called for example "Screenshots" and should be subimtted with your pull request. So that, you can write it as `![Alt Text](Screenshots/1.png)`